### DISCUSSION TITLE
New Film Format: Eastman EXR 500T 5298

### DISCUSSION THREAD

**Discussion by paulhopry** [videovillage/community#150] [Ideas]
I'd like to request the Eastman EXR 500T Filmstock to replicate similar looks to Heat 1995!

**Comment by wilg** [comment]
I think the big issue here is that we currently don't have a good way of profiling film stocks that are no longer made. But maybe there is a sneaky way to source the stock?

**Reply by paulhopry** [reply]
Oh damn, it’s not getting made anymore? What a shame. I loved that look and ive been searching for months for something that can replicate it. But sadly, there are no emulations for it, no powergrades, no nothing. Sorry for asking but what exactly do you mean by sourcing the stock?

**Reply by wilg** [reply]
To accurately add a stock to Filmbox, it requires us to purchase the film, shoot tests with it, get it developed, etc. It looks like this film stopped being made in 2003. https://filmcolors.org/timeline-entry/17423/ Which is a shame! I totally agree having these classic looks would be really cool.

**Reply by paulhopry** [reply]
I just found this: https://www.worthpoint.com/worthopedia/vintage-eastman-kodak-exr-500t-5298-479535419
Is this worth a look into?

**Reply by paulhopry** [reply]
https://www.ebay.de/itm/256133976590?mkcid=16&mkevt=1&mkrid=707-127634-2357-0&ssspo=KNPLBKY8T8O&sssrc=4429486&ssuid=&var=&widget_ver=artemis&media=COPY
Another offer, straight from germany, where I’m from. 300 Meters of Film, re-canned. What do you think?

**Reply by wilg** [reply]
We don't have the bandwidth to add new stocks right now, but this is very good to know!
